_translations/ja.yml

```diff
@@ -244,6 +244,7 @@ ja:
     walletelectrum: "Electrumの特徴は、低リソース、高速、シンプルな点です。ビットコインシステムの最も複雑な部分を扱うリモート・サーバーを使用しており、秘密の復元フレーズを使用してビットコイン・ウォレットを復元する事ができます。"
     walletmsigna: "mSIGNAは高性能かつ簡単に使用できるウォレットで、高速性、シンプルさ、企業レベルの拡張性、セキュリティーの強さが特徴です。これは、BIP32、マルチシグトランザクション、オフラインストレージ、マルチデバイス間の同期、暗号化された電子的および紙面によるバックアップをサポートしています。"
     walletbitcoinwallet: "ビットコインウォレットは、使用が簡単で信頼性があり、安全かつ高速な取引ができます。そのビジョンは、非中央集権化とゼロトラストにあります。つまり、ビットコインの運用に、中央集権的なサービスは必要ないと言う事です。このアプリは、技術的な知識を持たない方にも適しています。"
+    walletedgewallet: "Edgeは高レベルのプライバシー、セキュリティ、非中央集権性を備える、とても使いやすく、Segwitと互換性のあるビットコインモバイルウォレットです。Edgeウォレットは自動的に暗号化やバックアップがなされ、Edgeサーバーが停止しても機能します。"
```

Comment:
Please revise or drop change to this file as per comment in: `_translations/en.yml`